SimTransport/PPSProtonTransport/src/BaseProtonTransport.cc

```diff
@@ -3,20 +3,20 @@
 #include "Utilities/PPS/interface/PPSUnitConversion.h"
 #include <CLHEP/Random/RandGauss.h>
 #include <CLHEP/Units/GlobalSystemOfUnits.h>
-#include <cctype>
 
 BaseProtonTransport::BaseProtonTransport(const edm::ParameterSet& iConfig)
     : verbosity_(iConfig.getParameter<bool>("Verbosity")),
```

Comment:
Shouldn't initial values be assigned here, with the same intent of initializing these class members as it was in the first version of this PR?